src/ethereum/frontier/spec.py

```diff
@@ -594,6 +655,15 @@ def validate_transaction(tx: Transaction) -> bool:
     """
     Verifies a transaction.
 
+    The gas in a transaction gets used to pay for the intrinsic cost of
+    operations, therefore if there is insufficient gas then it would not
+    be possible to execute a transaction and it will be declared invalid.
+
+    Additionally, the nonce of a transaction must not equal or exceed the
+    limit defined in `EIP-2681 <https://eips.ethereum.org/EIPS/eip-2681>`_.
+    A nonce greater than or equal to 2**64-1 would be too large and cost an
```

Comment:
This is a bit unclear. The key point is that the limit has no impact is practice, because sending `2**64-1` transactions is implausible. It's not strictly impossible though, `2**64-1` transactions is the entire capacity of the Ethereum blockchain at 2022 gas limits for a little over 22 years.